unbricking.md

```diff
@@ -0,0 +1,193 @@
+# G-010S-A Unbricking
+
+If anything goes wrong during an update the SFP module can be unbricked through
+a serial console.
+
+## Preparation / Unlock Bootloader
+<a name="preparation"></a>
+
+By default the bootloader is locked and will not allow interrupting the boot.
```

Comment:
I think we should make clear that this "unlock" (in fact activating the UART lines) might require the pin 6 mod on adapters where it wasn't required (since the pin 6 was set to low by the preboot command, now it serves as TX).

Reply:
I have not seen an issue myself (have had the adapters in 3 different SFP ports on different devices). I have also not read it about issues after enabling the serial console anywhere.  But a warning about the possibility might be a good idea...

Reply:
The stock "closed" config has `gpio clear 100` as `preboot`, the only possible reasons would be :
 - for devices which would provide the initial 3.3V, and do not cut it if when pin 6 is briefly high (initial boot traces), but would if it was used as TX for longer
  - for devices which always provides the 3.3V, but require pin 6 to be low later to use the module

BCM57810S cards require the mod on either the card or the pin6 else the 3.3V is never turned on.  
IIRC, Mellanox ConnectX-2 cards always provide the 3.3V (and probably don't care about pin 6 state).

Reply:
added warning